src/query/storage/promremote/storage_test.go

```diff
@@ -135,13 +135,14 @@ func TestWrite(t *testing.T) {
 
 func TestDataRace(t *testing.T) {
 	fakeProm := promremotetest.NewServer(t)
+	scope := tally.NewTestScope("data_race", map[string]string{})
```

Comment:
We should remove the file scope var `scope` to force each test case to use its own scope. Otherwise, it's too easy to mix counters from two test cases.

Reply:
sg, let me do that